docs/resources/repository.md

```diff
@@ -63,9 +93,11 @@ resource "github_repository" "example" {
 - `archive_on_destroy` (Boolean) Set to 'true' to archive the repository instead of deleting on destroy.
 - `archived` (Boolean) Specifies if the repository should be archived. Defaults to 'false'. NOTE Currently, the API does not support unarchiving.
 - `auto_init` (Boolean) Set to 'true' to produce an initial commit in the repository.
+- `custom_property` (Block Set) Custom properties for the repository. (see [below for nested schema](#nestedblock--custom_property))
 - `default_branch` (String, Deprecated) Can only be set after initial repository creation, and only if the target branch exists
 - `delete_branch_on_merge` (Boolean) Automatically delete head branch after a pull request is merged. Defaults to 'false'.
 - `description` (String) A description of the repository.
+- `exclusive_custom_properties` (Boolean) Whether this resource exclusively manages all custom properties. If true, properties not defined in custom_property blocks will be removed. If false, only properties defined in custom_property blocks will be managed.
```

Comment:
Document the default? e.g. "Defaults to true; if set to 'false' … allowing collaborative management of custom property settings …"